src/validation/custom.validation.ts

```diff
@@ -29,3 +30,13 @@ export const checkDate = (errMsg: string, day: string = "", month: string = "",
   }
   return true;
 };
+
+export const checkDateIsNotFuture = (errMsg: string, day: string = "", month: string = "", year: string = "") => {
```

Comment:
I'd change the function name to checkDateIsInPast as today's date will throw an error message even though it is not in the future.